File: programme/even-or-odd-number/even-or-odd.go
// Write a programme to check a number even or odd

package main

import (
	"fmt"
	"strconv"
)

func main() {
	var str string

	fmt.Print("Input Number : ")
	fmt.Scan(&str)

	num, err := strconv.Atoi(str)
	if err != nil {
		fmt.Printf("Input error: %s", str)
		return
	}

	var evenOrOdd string
	if num%2 == 0 {
		evenOrOdd = "Even"
	} else {
		evenOrOdd = "Odd"
	}

	fmt.Printf("\nOutput : %s Number\n", evenOrOdd)
}

File: programme/even-or-odd-number/even-or-odd.py
# Write a programme to check a number even or odd
# n = 3 -> Odd Number, n=2 -> Even Number

def even_or_odd(num):
    return "Even" if num % 2 == 0 else "Odd"


n = int(input("Input  : "))
print("Output :", even_or_odd(n))
